Public/New-HTMLTableCondition.ps1

```diff
@@ -3,7 +3,7 @@ function New-HTMLTableCondition {
     param(
         [alias('ColumnName')][string] $Name,
         [alias('Type')][ValidateSet('number', 'string')][string] $ComparisonType,
-        [ValidateSet('lt', 'le', 'eq', 'ge', 'gt')][string] $Operator,
+        [ValidateSet('lt', 'le', 'eq', 'ge', 'gt', 'includes')][string] $Operator,
```

Comment:
Does includes act like contains? Should we use contains instead to use "PowerShell" convention?

Reply:
Whatever you think is best.

Reply:
I would like you to be part of this :-) It's your PR. I am pretty weak when it comes to JS/Jquery and mostly I don't know what I'm doing. I do appreciate people helping out and I honestly ask about other people opinions. What is the usage of that command? Can you provide some simple example?

Reply:
Sure, I'm keen too! I'm even more nub at JS. 

`-contains` in posh can be used against a collection whereas I think `includes()` method is just a against a string. However they are similar in the sense that

> "Hello world" -contains "Hello"

and

> "Hello World".includes("Hello")

would both return true.

I'm not sure what you mean by `Should we use contains instead to use "PowerShell" convention?` though. Is the idea that we just use posh to check `data[$ConditionHeaderNr]`? If so I'm not sure how you would get the value of `data`, although it would be really good if we could do it in posh because I could totally add in full regex support, just not sure how we could get `data` in to posh.

Reply:
I mean `'lt', 'le', 'eq', 'ge', 'gt', 'includes'`. Shouldn't it be `'lt', 'le', 'eq', 'ge', 'gt', 'contains'` ? Makes more sense to use it that way right?

Reply:
Agreed, I've updated the PR.